### PR TITLE
Add More Loadout Options

### PR DESCRIPTION
**Pull request by Haltell**
Added new loadout options for Captain, Chemist, Passenger, and Security.

Added Filled Jack Boots for security loadouts.

## About the PR

Added some extra loadout options from DeltaV/Nyanotrasen files.
Open to any other reasonable options for loadouts.

Current List of Additions:
Scarfs added to most roles, with the following exceptions:
- Assistant Roles
- Captain
- Chef
- Mantis
- Prison Guard
- Station Specific Jobs (except Corpsman)

Command:
- Captain & HoP's Inspection Gloves
- Captain & HoP's Brown, White, Lace Up, and Winter Shoes
- HoP's Outer Clothes
- HoP's Uniforms
- Captain's Beret
- All of Command has their winter boots as options

Medical:
- Paramedic Winter Boots given to Paramedic
- Stethoscopes given to CMO, Doctors, and Paramedics
Chemist:
- Heavy Nitrile Gloves
- Chemical Resistant Apron
- Enclosed Shoes
- Replaced Medical Winter Boots with Chemist Variant

Bartender:
- Changed glasses options. Added beer goggles.

Passenger:
- Blue, Red, Purple Casual Jumpsuit/skirt
- Yellow Skirt with Rose Hoodie
- Turquoise Hoodie and Shorts

Engineering:
- Removed requirement for hats
- Hats given to Atmospherics

Epistemics:
- Roboticist Labcoat for Scientists

Security:
- Jackboots
- Durathread Vest for Officers
- Red Beret for Officers and Cadets
- Blue Jumpsuit/skirt for Officers

Logistics:
- LO's DeltaV Beret
- Salvage Cloak + Miner Boots given to Salvage

## Why / Balance

Extra choices for characters to spend more time RPing and less time walking to, and standing in front of a clothes vending machine.

## Media

image
image
image
image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

<!--
Make sure to take this Changelog template out of the comment block in order for it to show up.
:cl:
--!>
- add: Nanotrasen uniform stores resotcked; many more options for your style!

### CODE REVIEW

File: Resources/Prototypes/Loadouts/role_loadouts.yml [hunk 1]
```diff
@@ -346,10 +346,10 @@
   groups:
   - MedicalMask
   - ChemistJumpsuit
-  - MedicalGloves
```
Comment (DangerRevolution) [inline Resources/Prototypes/Loadouts/role_loadouts.yml:349]: This needs to be commented out, and replaced with Chemist gloves *or* this needs to have a comment saying "Delta-V : Replaces MedicalGloves/Shoes with CHemistGloves/Shoes"

File: Resources/Prototypes/Loadouts/role_loadouts.yml [hunk 2]
```diff
@@ -346,10 +346,10 @@
...
+  - ChemistGloves # DeltaV
   - ChemistBackpack
   - ChemistOuterClothing
-  - MedicalShoes
+  - ChemistShoes # DeltaV
```
Comment (DangerRevolution) [inline Resources/Prototypes/Loadouts/role_loadouts.yml:381]: This needs to be commented out, and replaced with Chemist gloves *or* this needs to have a comment saying "Delta-V : Replaces MedicalGloves/Shoes with CHemistGloves/Shoes"

**Comment by DangerRevolution** [comment]
Changelog should also mention that Chemists now start with Chemist equivalents of shoes & gloves instead of Medical versions

**Comment by rosieposieeee** [comment]
red beret for all of sec or riot

**Comment by Adrian16199** [comment]
Give a red scarf to security, blue scarf to medical, green scarf to botany, yellow scarf to engineering, brown to logistics. No matter the role, just to all of them to said department.
Then this will be a good PR.

**Comment by Timemaster99** [comment]
I made a list in the discord of other things requiring attention in loadouts. Since you are open to further additions, I'll paste it here.

- HoP is missing their turtleneck, mess uniform, and their choice between durathread vest, armored coat, or jacket
- LO has the Wizden logistic's officer beret instead of ours
- MG's cloak sprite got overwritten by the Wizden one 
- Sec jobs have the two deprecated armor vests as options, and do not have the durathread vest
- Scientist is missing the Roboticist lab coat (The role has a roboticist jumpsuit option)
- Bartender is missing their beer goggles
- Musician has sunglasses by default in roles/musician.yml, so none of the universal glasses options work

File: Resources/Prototypes/Loadouts/role_loadouts.yml (hunk 1 above)
Comment (Haltell) [reply Resources/Prototypes/Loadouts/role_loadouts.yml:349]: Thanks, just fixed up all three of these things :)

**Comment by WarMechanic** [comment]
> Sec jobs have the two deprecated armor vests as options, and do not have the durathread vest

I don't think this is that bad, from my understanding people like the additional cosmetic variety that they provide.

**Comment by WarMechanic** [comment]
A change that I've been wanting to make is to make less experienced job uniforms available to more experience / command jobs:
For example, both station engineer and atmos tech gain access to yellow jumpskirts. Chief engineer would get access to station engineer, atmos tech and yellow jumpskirt loadout options.
I could do this in a separate PR. but I'm happy for you to do it here.

**Comment by Haltell** [comment]
> A change that I've been wanting to make is to make less experienced job uniforms available to more experience / command jobs: For example, both station engineer and atmos tech gain access to yellow jumpskirts. Chief engineer would get access to station engineer, atmos tech and yellow jumpskirt loadout options. I could do this in a separate PR. but I'm happy for you to do it here.

I'd like to see it too! But it more or less depends on if the people approving this want those options in game. It's otherwise a fairly simple addition of just adding more lines, but we've had issues in the past of heads trying to hide themselves and practically abandon the station during nukies, or similar.

**Comment by Timemaster99** [comment]
> > Sec jobs have the two deprecated armor vests as options, and do not have the durathread vest
> 
> I don't think this is that bad, from my understanding people like the additional cosmetic variety that they provide.

They were replaced by the durathread vest long ago for balancing, given their different armor values. They were consequently removed from maps and lockers. Discussion would be required to keep them, since they are currently unobtainable anywhere besides loadouts

**Comment by DangerRevolution** [comment]
> > > Sec jobs have the two deprecated armor vests as options, and do not have the durathread vest
> > 
> > 
> > I don't think this is that bad, from my understanding people like the additional cosmetic variety that they provide.
> 
> They were replaced by the durathread vest long ago for balancing, given their different armor values. They were consequently removed from maps and lockers. Discussion would be required to keep them, since they are currently unobtainable anywhere besides loadouts

The new vests are resprites because Wizden use the ugliest sprites imaginable, and I imagine no one has bothered updating the armour stats on them. It was never removed as a change exclusively to armour values.

**Comment by Timemaster99** [comment]
> > > > Sec jobs have the two deprecated armor vests as options, and do not have the durathread vest
> > > 
> > > 
> > > I don't think this is that bad, from my understanding people like the additional cosmetic variety that they provide.
> > 
> > 
> > They were replaced by the durathread vest long ago for balancing, given their different armor values. They were consequently removed from maps and lockers. Discussion would be required to keep them, since they are currently unobtainable anywhere besides loadouts
> 
> The new vests are resprites because Wizden use the ugliest sprites imaginable, and I imagine no one has bothered updating the armour stats on them. It was never removed as a change exclusively to armour values.

They would have changed the sprites instead of making new prototypes, I would strongly presume. They also do not have the same armor values. Here's some images
image
image
image
image

**Comment by Haltell** [comment]
> > > > Sec jobs have the two deprecated armor vests as options, and do not have the durathread vest
> > > 
> > > 
> > > I don't think this is that bad, from my understanding people like the additional cosmetic variety that they provide.
> > 
> > 
> > They were replaced by the durathread vest long ago for balancing, given their different armor values. They were consequently removed from maps and lockers. Discussion would be required to keep them, since they are currently unobtainable anywhere besides loadouts
> 
> The new vests are resprites because Wizden use the ugliest sprites imaginable, and I imagine no one has bothered updating the armour stats on them. It was never removed as a change exclusively to armour values.

Do you want me to comment them out quickly, as well?

**Comment by Timemaster99** [comment]
Chemist also requires their formal jumpsuit and tie

**Review by Axionyxx — APPROVED** [review]
Looks good to me, but I'd prefer someone else to read over them too

**Comment by therealDLondon** [comment]
*rubber stamp sound effect*
Headmin Approv!

**Pull request merged**